src/EditorFeatures/VisualBasicTest/GenerateOverrides/GenerateOverridesTests.vb

```diff
@@ -0,0 +1,39 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings
+Imports Microsoft.CodeAnalysis.GenerateOverrides
+Imports Microsoft.CodeAnalysis.PickMembers
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.GenerateOverrides
+    Public Class GenerateOverridesTests
+        Inherits AbstractVisualBasicCodeActionTest
+
+        Protected Overrides Function CreateCodeRefactoringProvider(Workspace As Workspace, parameters As TestParameters) As CodeRefactoringProvider
+            Return New GenerateOverridesCodeRefactoringProvider(DirectCast(parameters.fixProviderData, IPickMembersService))
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateOverrides)>
+        Public Async Function Test1() As Task
+            Await TestWithPickMembersDialogAsync(
```

Comment:
Consider tests for:

1. Finalizers.
2. Events.
3. Properties.
4. A negative test if there's nothing to generate.